sidebars.js

```diff
@@ -898,6 +898,11 @@ module.exports = {
             "self-managed/operational-guides/backup-restore/modeler-backup-and-restore",
           ],
         },
+        {
+          type: "doc",
+          label: "Configure components in Helm charts",
```

Comment:
Let's reduce that name to `Configure components` and mention Helm inside the page.

Reply:
I reduced this name